File: Unidad 3/Java/Programa19U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa19u3;

import java.util.Scanner;
public class Programa19U3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Ingrese la cantidad de numeros (N): ");
        int n = sc.nextInt();
        int igualCero = 0,menorCero = 0,mayorCero = 0;

        for (int i = 0; i < n; i++) {
            System.out.print("Ingrese el numero " + (i + 1) + ": ");
            double numero = sc.nextDouble();

            if (numero == 0) {
                igualCero++;
            } else if (numero > 0) {
                mayorCero++;
            } else {
                menorCero++;
            }
        }

        System.out.println("Numeros iguales a cero: " + igualCero);
        System.out.println("Numeros mayores a cero: " + mayorCero);
        System.out.println("Numeros menores a cero: " + menorCero);

    }
}

File: Unidad 3/C#/Programa19U3.cs
// See https://aka.ms/new-console-template for more information

using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Ingrese la cantidad de numeros (N): ");
            int n = Convert.ToInt32(Console.ReadLine());
            int igualCero = 0, menorCero = 0, mayorCero = 0;

            for (int i = 0; i < n; i++)
            {
                Console.WriteLine("Ingrese el numero " + (i + 1) + ": ");
                double numero = Convert.ToDouble(Console.ReadLine());

                if (numero == 0)
                {
                    igualCero++;
                }
                else if (numero > 0)
                {
                    mayorCero++;
                }
                else
                {
                    menorCero++;
                }
            }

            Console.WriteLine("Numeros iguales a cero: " + igualCero);
            Console.WriteLine("Numeros mayores a cero: " + mayorCero);
            Console.WriteLine("Numeros menores a cero: " + menorCero);
        }
    }
}